indico/modules/events/sessions/models/blocks.py

```diff
@@ -142,6 +142,14 @@ def can_manage_attachments(self, user):
     def can_edit_note(self, user):
         return self.session.can_edit_note(user)
 
+    @property
+    def start_dt(self):
+        return self.timetable_entry.start_dt if self.timetable_entry else None
+
+    @property
+    def end_dt(self):
+        return self.timetable_entry.start_dt + self.duration if self.timetable_entry else None
```

Comment:
You already have a property that wraps the timetable entry access and I'd say it's simpler like this:

    return self.start_dt + self.duration if self.start_dt is not None else None

Anyway, I just  saw we do the same thing you did in Contribution so we can also keep it consistent. In the end it doesn't really matter